app/src/main/java/com/framgia/quangtran/music_42/ui/genre/GenreActivity.java

```diff
@@ -21,11 +23,19 @@
 public class GenreActivity extends AppCompatActivity implements GenreContract.View {
     private static final int OFFSET = 1;
     public static final int LIMIT = 20;
-    private String mApi;
-    private static final String BUNDLE_GENRE = "genre";
+    private static final String BUNDLE_GENRE = "BUNDLE_GENRE";
```

Comment:
```suggestion
    private static final String BUNDLE_GENRE = "com.framgia.quangtran.music_42.ui.genre.BUNDLE_GENRE";
```
convention 2.10 for Intent